### ISSUE TITLE
Enketo: restyle all of the things

### ISSUE THREAD

**Issue by browndav** [medic/medic-webapp#1204]
Currently, the Enketo UI/UX and CSS isn't ideal. On the mid-iteration call, we talked about (i) restyling the report creation and report editing screens pre-demo; (ii) removing/resizing some of the buttons and making disabled/enabled states clearer; and (iii) potentially getting rid of the modal entirely and embedding Enketo in the right pane.

Items (i) and (ii) would be nice to have done pre-demo; item (iii) is probably discussion-only at this point.

**Comment by garethbowen** [comment]
I've done some style work for the report creation process in #1197 .

**Comment by alxndrsn** [comment]
It's not clear if (i) or (ii) are sufficiently improved to be considered done, but (iii) needs clear direction on what we're actually aiming for before work can start.

**Comment by browndav** [comment]
Waiting on a mockup.

**Comment by abbyad** [comment]
A new issue was created for item (iii): #1248

I think we can close this one and open a new one for further styling if needed.

**Comment by garethbowen** [comment]
Agreed.
